requirements.txt

```diff
@@ -1,6 +1,7 @@
 APScheduler==3.9.1
 beautifulsoup4==4.11.1
 boto3==1.23.6
+botocore~=1.26.10
```

Comment:
Since `boto3` is tightly coupled with `botocore` I would recommend keeping their pins in sync. As an example `boto3` version [1.23.10 ](https://github.com/boto/boto3/commit/dfea856b0a9ba385c7a9eda7d2a58e7cf0c16f2b) bumped the required minimum version of `botocore` to this version (1.26.10).

Reply:
addressed in https://github.com/cisagov/con-pca-api/pull/632/commits/18bbaccdc694efa1742e00419b255a30bd7fd401